fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -326,6 +343,14 @@ class DDTeamCollectionImpl {
 
 				for (int i = 0; i < randomTeams.size(); i++) {
 					int64_t loadBytes = randomTeams[i]->getLoadBytes(true, req.inflightPenalty);
+					if (req.storageQueueAware) {
```

Comment:
This `if` block is redundant, since all the randomTeams have already meet the condition in line 319~327